examples/thumbs.rs

```diff
@@ -0,0 +1,69 @@
+//! Generate jpg thumbnails in parallel
+//!
+//! From <https://rust-lang-nursery.github.io/rust-cookbook/concurrency.html#generate-jpg-thumbnails-in-parallel>
+
+extern crate image;
+#[macro_use]
+extern crate quicli;
+extern crate structopt;
+
+use quicli::prelude::*;
+
+use std::path::{Path, PathBuf};
```

Comment:
Maybe you are right (#20), @vitiral, and we should re-export some functions for dealing with paths.